app/views/passengers/_form.html.haml

```diff
@@ -15,17 +15,14 @@
         = form.form_group :registration_status,
           label: { text: 'Registration Status' },
           help: "You can select 'pending' if the passenger is temporary but has no verifying information yet" do
-          = form.radio_button :active_status, 'pending', class: 'form-check-input'
-          = form.radio_button :active_status, 'active', class: 'form-check-input'
-          = form.radio_button :active_status, 'archived', class: 'form-check-input'
+          = form.radio_button :registration_status, 'pending', class: 'form-check-input'
+          = form.radio_button :registration_status, 'active', class: 'form-check-input'
+          = form.radio_button :registration_status, 'archived', class: 'form-check-input'
```

Comment:
What are these changes doing here?

Reply:
I must have branched from my last branch instead of master (or accidentally made some changes in master as well as that branch).

Reply:
Remember `git add -i` and `git add --patch` are your friends. Even `git diff`